README.md

```diff
@@ -1 +1,9 @@
-# SLIV
\ No newline at end of file
+# SLAM Landmarks Interactive Visuailizer
+
+The aim of the project is to create a powerful desktop application to make debugging of SLAM frontend algorithms easier.
+
+This tool will provide following features:
+
+* Visualization of landmarks of three types(points, lines, planes) on top of images
```

Comment:
1. space before bracket 
2. on top of images -- not clear what images, let's either remove, either specify that "on top of images on which SLAM algorithm supposed to be working"